service/awsclusterconfig/v1/resource_set.go

```diff
@@ -125,6 +157,15 @@ func NewResourceSet(config ResourceSetConfig) (*framework.ResourceSet, error) {
 	return resourceSet, nil
 }
 
+func toClusterGuestConfig(obj interface{}) (v1alpha1.ClusterGuestConfig, error) {
+	awsClusterConfig, err := key.ToCustomObject(obj)
+	if err != nil {
+		return v1alpha1.ClusterGuestConfig{}, microerror.Mask(err)
+	}
+
+	return key.ToClusterGuestConfig(awsClusterConfig), nil
+}
```

Comment:
This function is passed to the `chartconfig` resource so it can handle `awsclusterconfig` CRs.